package.json

```diff
@@ -9,9 +9,9 @@
   "license": "MIT",
   "workspaces": [
     ".github/workflows/scripts",
-    "shared/*",
     "packages/govuk-frontend",
     "packages/govuk-frontend-review",
+    "shared/*",
```

Comment:
For `npm run build --workspaces` we need **govuk-frontend** to build before shared packages